css/index.css

```
@@ -0,0 +1,179 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto";
+  font-family: 'Source Sans Pro';
```

Estas cargando dos fuentes, pero solo se usa la ultima que escribes. Ademas, es buena practica colocar fuentes de backup.
```suggestion
  font-family: 'Source Sans Pro', Arial, sans-serif;
```